cpp/harvest/marc_issn_lookup.cc

```diff
@@ -43,6 +44,7 @@ struct SubFieldInfo {
     std::string t_;
     std::string w_;
     std::string x_;
+    std::vector<std::string> issns; // this is for reference issn from main
```

Comment:
Since it's a class property, the variable name should still end with an underscore => `issns_`